jenkins-design-language/less/components/button.less

```diff
@@ -129,6 +129,10 @@ button, .btn, .btn-default, .btn-primary, .btn-secondary, .btn-danger, a.btn-lin
 button, .btn, .btn-default, .btn-primary, .btn-secondary {
     &.monochrome {
         .btn-styles(transparent, @gray-light, @text-color, @gray-lighter, @gray-lighter);
+
+        svg {
+            fill: @text-color !important;
```

Comment:
I did this because of the monochrome icons I talked about during our meeting, it is just to make sure that all svgs that are children of elements with ".monochrome" have the color of the text, I can't increase specificity in this case because the fill rule that needs to be beaten is inline, only important can trump that. This could be removed if all the components that have monochrome icons correctly sends the fill color, I added it because the way some components are made right now make this really difficult right now, can't remember which one right now